htdocs/societe/card.php

```diff
@@ -3339,3 +3359,46 @@ function init_check_no_email(input) {
 // End of page
 llxFooter();
 $db->close();
+
+?>
+
+<script>
+// Function to retrieve VAT details from the Greek Ministry of Finance GSIS SOAP web service
+function GRVAT(a, u, p, myafm) {
+  var afm = a.replace(/\D/g, ""); // Remove non-digit characters from 'a'
+
+  $.ajax({
+	type: "GET",
+	url: "/societe/checkvat/checkVatGr.php",
+	data: { u: u, p: p, myafm: myafm, afm: afm }, // Set request parameters
```

Comment:
There is no need to propagate user and pass as parameter because there are global constant.